Makefile

```diff
@@ -23,5 +24,8 @@ run-%: bin/% $$(wildcard %.c) $$(wildcard %.cpp)
 	scp -P $(SSHPORT) $(word 2,$^) bin/$(<F) root@127.0.0.1:/root
 	ssh -p $(SSHPORT) root@127.0.0.1 -t '/root/$(<F)'
 
+c-format: 
```

Comment:
I think we should either call this `cformat` (I always forget `-` characters) or `clang-format` (on the basis that that's the actual tool being run). I have no preference for which we go for.

Reply:
Agreed.

Reply:
bors try

Reply:
FIXED in d1bc2bb.

Reply:
But I believe I have made a mistake. Sorry.